Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/LambdaBootstrapBuilder.cs

```diff
@@ -449,14 +451,28 @@ public LambdaBootstrapBuilder UseBootstrapHandler(LambdaBootstrapInitializer lam
             return this;
         }
 
+        /// <summary>
+        /// Configure Lambda Bootstrap options
+        /// This is for testing purposes and should not be used in production.
```

Comment:
is there any way to prevent this from being used in production besides just having a comment?

Reply:
I don't think we want to prevent people from using it. If someone has a use case and this fits it, then why not. However, for Lambda's production environment, the value is passed using env vars. So we're adding a note for people to see that this is meant for testing purposes. I don't know if Norm has different thoughts on it.